### ISSUE TITLE
QEditor hyperlink button click errors

### ISSUE THREAD

**Issue by rvyas** [quasarframework/quasar#5134]
**Describe the bug**
1. Recently updated to quasar 1.1.4. In QEditor, when I  hit the hyperlink button, it creates the link however displays error in console.
2. QEditor's hyperlink box does not display properly when wrapped in QField.

**Codepen/jsFiddle/Codesandbox (required)**
https://codepen.io/anon/pen/aeBbMj?editors=1011

**To Reproduce**
1. Go to https://codepen.io/anon/pen/aeBbMj?editors=1011
2. Select text
2. Click on 'hyperlink button' or Ctrl+L
3. Check console for error

**Expected behavior**
It should create hyperlink without errors.

**Screenshots**
```
---> <QEditor>
       <QField>
         <Root>"

TypeError: Cannot read property 'selectNodeContents' of undefined
    at VueComponent.<anonymous> (editor-caret.js:223)
    at Array.<anonymous> (vue.runtime.esm.js:1980)
    at flushCallbacks (vue.runtime.esm.js:1906)
```

**Comment by iperiago** [comment]
Bumping this and adding the error occurs even if the QEditor component is not wrapped in a QField. The link does get created but the error is annoying. Looked briefly at the QEditor component's editor-caret.js file and couldn't really figure out why the code triggering the error is needed. It seems creating the link empties the selection which is then queried while empty, triggering the error. Maybe someone can help.

**Comment by rstoenescu** [comment]
Fix will be available in "quasar" v1.5.8
